src/main/java/Main.java

```diff
@@ -15,4 +19,19 @@ public static void main(String[] args) {
 (по 3-4 варианта входных данных).
          */
     }
+
+    public int[] arrayFilter(int[] array) {
+        List<Integer> list = Arrays.stream(array).boxed().collect(Collectors.toList());
+        int position = list.lastIndexOf(4);
+        if(position == -1){
+            throw new RuntimeException("Array does not contain 4");
+        }
+        return Arrays.copyOfRange(array,position + 1,array.length);
+    }
+
+    public boolean isArrayContains(int[] array){
+        if(Arrays.stream(array).anyMatch(x -> x == 4) && Arrays.stream(array).anyMatch(x -> x == 1)){
```

Comment:
`return Arrays.stream(array).anyMatch(x -> x == 4) && Arrays.stream(array).anyMatch(x -> x == 1);`

Reply:
repaired in 4ece734